logs.txt

```diff
@@ -0,0 +1,57 @@
+ * Serving Flask app "todo_app/app" (lazy loading)
+ * Environment: development
```

Comment:
Worth adding this to your .gitignore, then delete the file and commit the changes.